mig/install/MiGserver-template.conf

```diff
@@ -187,6 +187,7 @@ mig_server_id = %(server_fqdn)s.0
 empty_job_name = no_grid_jobs_in_grid_scheduler
 notify_protocols = email
 smtp_server = __SMTP_SERVER__
+storage_protocols = __STORAGE_PROTOCOLS__
```

Comment:
I'd like to add a few comment lines of `storage_protocols` docs and probably move the option a bit around in the file as a follow-up PR. Adding the follow-up marker to remember.

Reply:
now handled in PR137 so removing the follow-up tag here.